src/SpaceTime/2d/Acceleration2.cs

```diff
@@ -0,0 +1,272 @@
+﻿using System;
+using Bearded.Utilities.Math;
+using OpenTK;
+
+namespace Bearded.Utilities.SpaceTime
+{
+    /// <summary>
+    /// A type-safe representation of a 2d directed acceleration vector.
+    /// </summary>
+    public struct Acceleration2 : IBackedBy<Vector2>, IEquatable<Acceleration2>
+    {
+        private readonly Vector2 value;
+
+        #region construction
+
+        /// <summary>
+        /// Creates a new instance of the Acceleration2 type.
+        /// </summary>
+        public Acceleration2(Vector2 value)
+        {
+            this.value = value;
+        }
```

Comment:
I'm not using empty lines between a lot of related methods (whether they be overloads, or a group of operators), basically at all, inside the same region (with a few exceptions, granted). Should I add empty lines everywhere?